### DISCUSSION TITLE
Can i use signoz without docker

### DISCUSSION THREAD

**Discussion by Yahire3388** [SigNoz/signoz#876] [Q&A]
Hi team,

What i had understood from the document and the research i had done on signoz is, even it is opensource but it supports docker application/containers only. Please correct me if i am wrong.

Can you please help me with any documentation if available for the intent of installation and setup of signoz in windows system, to monitor web application running on tomcat server like we can monitor using opentelemetry java agent in few simple steps by configuring setenv.bat file and to export the tracing details to database or to exporters.

Thanks in advance.

**Comment by pranay01** [comment]
hey @Yahire3388 - yup you are right. Primary way to install is via docker containers. Either directly or through k8s. Do you have any concerns/restrictions on using docker?

Though we don't officially support Windows, you can try setting up the docker in windows.
